src/com/android/volley/toolbox/NetworkMonitor.java

```diff
@@ -57,7 +60,7 @@ public static void add(NetworkResponse response, ImageRequest request) {
             if (bandwidthKBPS < BANDWIDTH_HYSTERETIC_LOWER) {
                 lowBandwidth = true;
                 log("NETMON", "Low-Bandwidth flag On");
-            } else if (bandwidthKBPS > BANDWIDTH_HYSTERETIC_UPPER) {
+            } else if (bandwidthKBPS > BANDWIDTH_HYSTERETIC_UPPER && (!mIsRoaming || !mForceLowBandwidthIfRoaming)) {
```

Comment:
the two are logically identical. since if ```!mIsRoaming``` is ```true``` then ```mIsRoaming``` is by default true and we don't need the extra conditional. 

I could also demorgan's law it to !(mIsRoaming && mForceLowBandwidthMode)